reactioncommerce/reaction
# Fix console error for products without variants

Resolve #3349 

PR resolve console error displayed when variant does not exist in the PDP. This actually happens because the product variant value is empty hence a check is placed to make sure variant exists before checking for `indexOf`

#### Test
- Create a new product
- Go to your database GUI and delete the variant instance of the new product
- Do not refresh the product grid page
- Visit the product grid and click on the product created above
- Observe the browser console to see no error displayed
